### ISSUE TITLE
Grouping objects - allow Left+Click to also de-select an object

### ISSUE THREAD

**Issue by rvanner** [TheGameCreators/GameGuruRepo#3019]
Control+Left click should both select and de-select objects. 

Use case: Rubber band a lot of objects and then use Control+LMB to de-select any objects you don't want in the group.

I class this as a bug because it should do this.

**Comment by ArgentArts** [comment]
I agree. We should be able to CTRL+Click to both ADD to and SUBTRACT from a group selection as it is pretty much a standard way of functioning in software.
